cas/softcas/softcas.go

```diff
@@ -112,6 +113,17 @@ func (c *SoftCAS) RevokeCertificate(req *apiv1.RevokeCertificateRequest) (*apiv1
 	}, nil
 }
 
+// CreateCertificateRevocationList will create a new CRL based on the RevocationList passed to it
+func (c *SoftCAS) CreateCertificateRevocationList(crl *x509.RevocationList) ([]byte, error) {
```

Comment:
I think we want to make it more extensible and add a `*apiv1.CreateCertificateRevocationListRequest` and `*apiv1.CreateCertificateRevocationListResponse`, although initially they will only have the CRL on the request and return a field named CRL too (or SignedCRL, ASN1Bytes, what do you think is better).

Reply:
Perhaps we can name everything `CreateCRL*`, `CreateCertificateRevocationListRequest` seems too long.